Dfe.Academies.Academisation.Data/ApplicationAggregate/ApplicationSchoolState.cs

```diff
@@ -53,7 +53,13 @@ public class ApplicationSchoolState : BaseEntity
 	public string? PartOfPfiSchemeType { get; set; }
 	public bool? PartOfPrioritySchoolsBuildingProgramme { get; set; }
 	public bool? PartOfBuildingSchoolsForFutureProgramme { get; set; }
+	// application pre-support grant
 
+	/// <summary>
+	/// either "To the school" or "To the trust the school is joining"
```

Comment:
If we have this comment, I think it should be on the Domain object, not the data entity

Reply:
However, see comment on domain object